app/models/cinstance.rb

```diff
@@ -110,16 +110,15 @@ def validate_plan_is_unique?
 
   validate :same_service, on: :update, if: :plan_id_changed?
 
-  APP_ID_FORMAT = /[\w-]+/.freeze
+  APP_ID_FORMAT = /[\x20-\x7E]+/
```

Comment:
Not freezing it?

Reply:
On purpose 😄 
![image](https://user-images.githubusercontent.com/11318903/60724632-4750b500-9f37-11e9-85fe-eb1c040eb848.png)

Reply:
wow, interesting 🤔 
![image](https://user-images.githubusercontent.com/11318903/60724807-a9111f00-9f37-11e9-9aae-afc952417619.png)